src/Core/Models/Api/Request/TwoFactorRequestModels.cs

```diff
@@ -238,7 +238,7 @@ public class TwoFactorWebAuthnDeleteRequestModel : SecretVerificationRequestMode
 
         public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
         {
-            foreach (var validationResult in Validate(validationContext))
+            foreach (var validationResult in base.Validate(validationContext))
```

Comment:
does this need to override base class? I am seeing warnings in VS for me.

Reply:
I'm not familiar with the particular warning, but it wants me to add `new` before IEnumerable.

Reply:
ok, `new` it is. Try that and verify it works still I guess.

Reply:
new will shadow the previous implementation, but I believe will only work if the type is specifically cast to `TwoFactorRequestModel`, I don't think that's what we want

Reply:
more than likely it'd be better to make the base class `virtual` and `override` here